installbuilder/datafiles/Base_DSC.data

```diff
@@ -351,9 +339,7 @@ if [ "$1" = "0" -o "$1" = "remove" -o "$1" = "purge" ]; then
     # Only to be removed for standalone DSC case 
     rm -f /etc/cron.d/dsc
 #endif
-    for prov in ${{PROVIDERS}}; do
-        rm -f $OMI_LIB/libMSFT_nx${prov}Resource.so
-    done
```

Comment:
where do we call remove module now to uninstall modules during package uninstall ?

Reply:
It's was already being done in the pre-uninstall section a little above this removed section